src/Sample.xoo

```diff
@@ -1,6 +1,7 @@
 public class Sample1 {
 
   private int i;
+  //TODO xoo XOO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://localhost:9000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://localhost:9000/project/issues?id=sample&branch=teryk-changes&issues=AWD5YMrVg9FTwNLAbP3x&open=AWD5YMrVg9FTwNLAbP3x)

[ISSUE_KEY]: AWD5YMrVg9FTwNLAbP3x